src/config.rs

```diff
@@ -245,7 +245,7 @@ impl ConfigWrapper {
         ))
     }
 
-    pub fn reload(&mut self) -> Result<(), CrustError> {
+    fn reload(&mut self) -> Result<(), CrustError> {
```

Comment:
It seems like making these methods private doesn't really have anything to do with TCP disable. Hence, I would put those changes to a separate commit and keep my commits atomic. The benefits of that is explained here: https://www.freshconsulting.com/atomic-commits/